src/coreclr/jit/hwintrinsicxarch.cpp

```diff
@@ -172,9 +180,16 @@ static CORINFO_InstructionSet lookupInstructionSet(const char* className)
             {
                 return InstructionSet_AVX;
             }
-            else if (strcmp(className + 3, "10v1") == 0)
+            else if (strcmp(className + 3, "10v") == 0)
```

Comment:
Doesn't this need to be:
```suggestion
            else if (strncmp(className + 3, "10v", 3) == 0)
```
?